### PR TITLE
Windup report comparison test

### PR DESCRIPTION
**Pull request by pranavgaikwad**
This adds a new subcommand to run windup and analyzer both with a test application and compare results of the analysis and print.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -11,20 +11,23 @@ import (
 
 	"github.com/fabianvf/windup-rulesets-yaml/pkg/conversion"
 	"github.com/fabianvf/windup-rulesets-yaml/pkg/execution"
+	"github.com/fabianvf/windup-rulesets-yaml/pkg/report"
```
Comment (fabianvf) [inline main.go:14]: Guess I should update these at some point, whoops

File: Dockerfile [hunk 2]
```diff
@@ -11,11 +13,19 @@ RUN go build -o windup-shim main.go
...
+ARG WINDUP=https://repo1.maven.org/maven2/org/jboss/windup/tackle-cli/6.2.4.Final/tackle-cli-6.2.4.Final-offline.zip
+RUN wget -qO /tmp/windup.zip $WINDUP \
+ && unzip /tmp/windup.zip -d /windup \
+ && rm /tmp/windup.zip \
+ && ln -s /windup/tackle-cli-*/bin/windup-cli /usr/bin/windup-cli
+
+RUN git clone https://github.com/windup/windup-rulesets.git -b 6.2.3.Final /windup-rulesets \
+  && git clone https://github.com/konveyor/example-applications /example-applications
+
+COPY --from=java-builder /usr/local/openjdk-11 /java-11-openjdk
```
Comment (shawn-hurley) [inline Dockerfile:28]: Wondering why we need this?

Comment (pranavgaikwad) [reply Dockerfile:28]: @shawn-hurley windup cli requires 11

**Comment by shawn-hurley** [comment]
Lurk to

**Pull request merged**